content/blog/2023/11/raw-policies.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Raw policies
+authors:
+  - Fabrizio Sestito
+date: 2023-11-03
+---
+
+We are happy to announce that Kubewarden 1.9.0 will introduce even more features requested by the community!
+
+In this blog post, we will introduce the new `Raw` policy type.
+
+## Kubewarden as a generic policy engine
+
+Raw policies allow policy authors to write and execute policies that are not necessarily related to Kubernetes.
+This means that Kubewarden can be used as a general-purpose policy engine.
+For instance, you can use Kubewarden to validate any type of artifact:
+configurations, Terraform plans, test coverage, static analysis
+or even deploy Kubewarden alongside your web application to validate domain-specific requests.
+
+## Key features
+
+Raw policies offer several key features:
+
+- Simple API: The Policy Server exposes the `/validate_raw` endpoint, enabling the validation or mutation of _arbitrary_ JSON documents against Kubewarden policies.
+
+- Language Agnostic: Raw policies can be written in any language that supports WebAssembly, such as Go, Rust, and OPA.
+
+- Versatility: Raw policies provide a high degree of flexibility, allowing policy authors to define policies tailored to their specific needs.
+
+- Standalone Execution: The Policy Server can be run outside Kubernetes as a standalone container if needed, which simplifies the deployment and execution of raw policies and the integration with other systems.
+
+## Using the validate raw endpoint
+
+Let's see the `/validate_raw` endpoint of a local Policy Server running on port `3000` in action:
+
+This request is accepted:
+
+{{<figure src="/images/raw_policy_accepted.gif" alt="raw policy accepted demo">}}
```

Comment:
The gif show a response where the `allowed` fields has the `false` value and a message telling why the request is not allowed. 

![image](https://github.com/kubewarden/kubewarden.io/assets/1514798/f8c9dfd6-229f-450a-827b-21cf5d904efd)